src/src/com/microsoft/aad/adal/AuthenticationDialog.java

```diff
@@ -200,7 +201,8 @@ public void showSpinner(final boolean status) {
                     public void run() {
                         if (mDialog != null && mDialog.isShowing()) {
                             ProgressBar progressBar = (ProgressBar)mDialog
-                                    .findViewById(R.id.com_microsoft_aad_adal_progressBar);
+                                    .findViewById(getResourceId(
+                                            "com_microsoft_aad_adal_progressBar", "id"));
```

Comment:
To fix reference issue with this component. It does lookup for given constant name and type from actual package that is using this library.